OpenTrons/opentrons-api
# feat(app): run cancelling modal for odd

closes RAUT-486

# Overview

Canceling run modal appears on the ODD when you 1) cancel a run from the ODD and click to confirm the cancel 2) cancel a run from the desktop app

# Test Plan

Unfortunately i am unable to properly test this:

- Start a run on the desktop app and cancel it, the touchscreen should show the "canceling run" modal
- Start a run and stop it on the touchscreen, when canceling, it should show the "canceling run" modal

# Changelog

- create new modal component for cancelling run called `CancelingRunModal` and create a test
- plug the modal into both `ConfirmCancelRunModal` and `RunningProtocol`, update tests

# Review requests

see test plan

# Risk assessment

low
